packages/core/src/Serialization/TransactionBody/ProposalProcedure/InfoAction.ts

```diff
@@ -22,6 +24,7 @@ export class InfoAction {
 
     // CDDL
     // info_action = 6
+    writer.writeStartArray(EMBEDDED_GROUP_SIZE);
```

Comment:
Great job @iccicci 🕵🏼 . I think we should extract this change (and the unit test update) to a separate commit as a fix. 

Also we should probably revert the changes in this commit:

https://github.com/input-output-hk/cardano-js-sdk/commit/8ec646a657df80bd0d43f2ea134cffda06e2a4b0

That commit was attempting to fix this issue but was totally wrong 😅. Your change is the correct fix